packages/skill-lesson/schemas/base-lesson-resource.ts

```diff
@@ -7,4 +7,20 @@ export const BaseLessonResourceSchema = z.object({
   slug: z.string(),
   description: z.nullable(z.string()).optional(),
   body: z.nullable(z.any().array().optional()),
+  solution: z
+    .nullable(
+      z.object({
+        _key: z.string(),
+        slug: z.nullable(z.string()).optional(),
+        videoResourceId: z.nullable(z.string()).optional(),
+        transcript: z.nullable(z.any().array()).optional(),
+        github: z
+          .object({
+            url: z.string(),
+          })
+          .optional()
+          .nullable(),
+      }),
+    )
+    .optional(),
```

Comment:
this can't happen. the entire purpose of this schema is to prevent this sort of application specific type.